pandas/tests/io/pytables/test_keys.py

```diff
@@ -9,15 +9,15 @@
     date_range,
 )
 from pandas.tests.io.pytables.common import (
-    ensure_clean_store,
-    tables,
+    tables
```

Comment:
It should be

```python
from pandas.tests.io.pytables.common import tables
```

See the [pre-commit](https://pandas.pydata.org/docs/development/contributing_codebase.html#contributing-pre-commit) section for how to run code checks. It will catch errors like this.